cabot/python/cabot/handle_v2.py

```diff
@@ -61,13 +63,15 @@ def __init__(self, realworld_use=True, event_listener=None, button_keys=[]):
         self.duration_single_vibration = 40
         self.duration_about_turn = 40
         self.duration_button_click = 5
+        self.duration_button_holddown = 10 # [ms] ?
```

Comment:
I know this unit is wired, but `1` is about 10ms.
There is a loop to execute `sleep(0.01)` for the number of the duration.